packages/imperative/src/profiles/handlers/CreateProfilesHandler.ts

```diff
@@ -41,6 +41,7 @@ export default class CreateProfilesHandler implements ICommandHandler {
             type: profileType,
             args: commandParameters.arguments,
             overwrite: commandParameters.arguments.overwrite,
+            disableDefaults: commandParameters.arguments.disableDefaults,
```

Comment:
Disclaimer: I can see that the precedent was already set, and I would not change your implementation at this time.

However, since commandParameters.arguments are already included in the createParms variable, does commandParameters.arguments.disableDefaults need to be passed in separately? Again, don't change it. I was just interested in your thoughts.

Reply:
That's a good point.
May we should deprecate overwrite instead of adding a new thing to the interface and just process them from the `args` options :yum:

Reply:
Created https://github.com/zowe/imperative/issues/403 as a reminder.
Thanks again @gejohnston